mmv1/third_party/terraform/website/docs/d/apphub_discovered_service.html.markdown

```diff
@@ -0,0 +1,45 @@
+---
+subcategory: "Apphub"
+description: |-
+  Get information about a discovered service.
+---
+
+# google\_apphub\_discovered_service
+
+Get information about a discovered service from its uri.
+
+
+## Example Usage
+
+
+```hcl
+data "google_apphub_discovered_service" "my-service" {
+  location = "my-location"
+  service_uri = "my-service-uri"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `project` - The host project of the discovered service.
+* `service_uri` - (Required) The uri of the service.
+* `location` - (Required) The location of the discovered service.
+
+## Attributes Reference
+
+In addition to the arguments listed above, the following computed attributes are exported:
+
+* `discovered_service` - Represents a network/api interface that exposes some functionality to clients for consumption over the network. A `discovered_service` object would contain the following fields:-
+
+  - `name` - Resource name of a Service. Format: "projects/{host-project-id}/locations/{location}/applications/{application-id}/services/{service-id}".
+
+  - `service_reference` - Reference to an underlying networking resource that can comprise a Service. A `service_reference` object would contain the following fields:-
+    - uri - The underlying resource URI.
+    - path - Additional path under the resource URI.
+
+  - `service_properties` - Properties of an underlying compute resource that can comprise a Service. A `service_properties` object would contain the following fields:-
+      - gcp_project - The service project identifier that the underlying cloud resource resides in.
```

Comment:
These will be displayed on the website, so I'm not sure if the indentation changes how it renders, but we will want these bullets to be consistent (for example, between service_reference and service_properties).

Reply:
done